connectors/src/connectors/microsoft/lib/content_nodes.ts

```diff
@@ -1,7 +1,9 @@
 import type { ContentNode } from "@dust-tt/types";
 
 import {
+  getDriveApiPath,
```

Comment:
nit: no need to tackle here, but might be worth tackling esp if some models have that issue. API is an acronym so we should write `getDriveAPIPath` :pray: